docs/edgeql/ddl/functions.rst

```diff
@@ -229,7 +142,8 @@ Change the definition of a function.
     # and <subcommand> is one of
 
       SET volatility := {'VOLATILE' | 'IMMUTABLE' | 'STABLE'} ;
-      RESET volatility
+      RENAME TO <newname> ;
```

Comment:
I'd move RENAME TO below RESET volatility.

Reply:
Fixed